src/commands/app/logs.ts

```diff
@@ -89,22 +95,24 @@ export default class AppLogs extends Command {
         this.debug(error.stack);
       }
 
-      const lastLog = logs[logs.length - 1];
+      const lastLog = logs[0];
 
-      if (lastLog && lastLog.datetime === 'Error') {
+      if (lastLog && lastLog[0] === 'Error') {
         // tslint:disable-next-line: no-console
         console.error(
-          new Errors.CLIError(`${lastLog.message}
+          new Errors.CLIError(`${lastLog[1]}
 Sorry for inconvenience. Please contact us.`).render()
         );
         process.exit(1);
       }
 
       if (lastLog) {
-        since = moment(lastLog.datetime).unix() + 1;
+        const unixTime = lastLog[0].slice(0, 10);
+        since = parseInt(unixTime) + 1;
       }
 
-      for (const log of logs) {
+      for (let i = logs.length - 1; i >= 0; i--) {
+        const log = logs[i];
```

Comment:
I think this is more readable:

```suggestion
      for (const log of logs.reverse()) {
```